apps/docs/src/app/examples/dialogs/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import {
+  HvButton,
+  HvDialog,
+  HvDialogActions,
+  HvDialogContent,
+  HvDialogTitle,
+} from "@hitachivantara/uikit-react-core";
+
+export default function Demo() {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <>
+      <HvDialog
+        open={open}
+        onClose={() => setOpen(false)}
+        maxWidth="sm"
+        fullWidth
+      >
+        <HvDialogTitle variant="error">Are you sure?</HvDialogTitle>
+        <HvDialogContent>This action cannot be undone.</HvDialogContent>
+        <HvDialogActions>
+          <HvButton variant="secondarySubtle" onClick={() => setOpen(false)}>
```

Comment:
for these destructive actions, it'd be nice if we `autoFocus` the non-destructive option
```suggestion
          <HvButton autoFocus variant="secondarySubtle" onClick={() => setOpen(false)}>
```